docs/howto/import.md

```diff
@@ -16,47 +16,27 @@ redirect_from:
 
 ## Zero-copy import
 
-### Importing using the lakeFS UI
+### Using the lakeFS UI
 
-#### Prerequisites
-
-lakeFS must have permissions to list the objects at the source object store,
-and in the same region of your destination bucket.
-
-lakeFS supports two ways to ingest objects from the object store without copying the data:
-
-1. [Importing using the lakeFS UI](#importing-using-the-lakefs-ui) - A UI dialog to trigger an import to a designated import branch. It creates a commit from all imported objects.
-1. [Importing using lakectl cli](#importing-using-lakectl-cli) - You can use the [`lakectl` CLI command](../reference/cli.md#lakectl) to create uncommitted objects in a branch. It will make sequential calls between the CLI and the server.
-
-#### Using the import wizard
-
-Clicking the Import button from any branch will open the following dialog:
-
-![Import dialog example configured with S3](../assets/img/UI-Import-Dialog.png)
-
-If it's the first import to the selected branch, it will create the import branch named `_<branch_name>_imported`.
-lakeFS will import all objects from the Source URI to the import branch under the given prefix.
-
-The UI will update periodically with the amount of objects imported. How long it takes depends on the amount of objects to be imported but will roughly be a few thousand objects per second.
-
-![img.png](../assets/img/ui-import-waiting.png)
-
-Once the import is completed, you can merge the changes from the import branch to the source branch.
-
-![img.png](../assets/img/ui-import-completed.png)
-
-
-### Importing using lakectl cli
+To import using the UI, lakeFS must have permissions to list the objects in the source object store.
+{: .note }
 
-The `lakectl` cli supports _import_ and _ingest_ commands to import objects from an external source.
+1. In your repository's main page, click the _Import_ button to open the import dialog:
+   ![img.png](../assets/img/UI-Import-Dialog.png)
+2. Under _Import from_, fill in the location on your object store you would like to import from.
+3. Fill in the import destination in lakeFS and a commit message.
+4. Once the import is completed, you can merge the changes from the import branch to the source branch.
```

Comment:
nit: this isn't step 4 but a comment to put underneath the list. Either that, or make numbered items explaining how to do the merge (or maybe link to the relevant doc page)

Reply:
Done